hardware.inc

```diff
@@ -145,6 +153,18 @@ DEF TACF_262KHZ EQU %00000001
 ; --
 DEF rIF EQU $FF0F
 
+DEF IFF_HILO   EQU %00010000 ; Transition from High to Low of Pin number P10-P13

```

Comment:
There are already covered by `IEF_*`

Reply:
Yeah fair enough. I'll remove them, and move/rename the bit numbers to the IE section.